### ISSUE TITLE
Bugs reporting

### ISSUE THREAD

**Issue by omarkamouh** [mitigation-controller/mitc#29]
**Is your feature request related to a problem? Please describe.**
Bugs reporting

**Describe the solution you'd like**
I would like to be able to report bugs directly from the user interface y having an option in the menu

**Describe alternatives you've considered**
...

**Additional context**
..

**Comment by mwakok** [comment]
@omarkamouh Would it be sufficient to have a link in the GUI that directs the user to the github issue template?

Reference: https://nl.mathworks.com/matlabcentral/answers/443574-how-to-link-to-a-web-address-or-a-local-pdf-file-from-app-designer

**Comment by omarkamouh** [comment]
@mwakok Yes that would be enough. I noticed however that you have to be a registered user to be able to report bugs. Can we allow unregistered users to report bugs? If not, I would use another platform to report bugs and issues that does not require registration.

**Comment by mwakok** [comment]
I had a brief look and it seems somebody has already created a solution called [gitreports](https://gitreports.com/).  I will take a look and see if it is useful for us.

**Comment by mwakok** [comment]
Another option would be to use: https://fire.fundersclub.com/
